asreviewcontrib/datatools/sample.py

```diff
@@ -0,0 +1,83 @@
+import argparse
+from pathlib import Path
+
+import pandas as pd
+from asreview import ASReviewData
+from asreview.data.base import load_data
+
+
+def _check_suffix(input_file, output_file):
+    # Also raises ValueError on URLs that do not end with a file extension
+    suffixes = [Path(input_file).suffix, Path(output_file).suffix]
+
+    set_ris = {".txt", ".ris"}
+    set_tabular = {".csv", ".tab", ".tsv", ".xlsx"}
+    set_suffixes = set(suffixes)
+
+    if len(set(suffixes)) > 1:
+        if not (set_suffixes.issubset(set_ris) or set_suffixes.issubset(set_tabular)):
+            raise ValueError(
+                "• Several file types were given; The input and the output file" 
+                "should be of the same type. "
+            )
+
+
+def sample(output_path, input_path, nr_records):
+    _check_suffix(input_path, output_path)
```

Comment:
I don't think it's necessary to check for the file type, since `asreview` will already check this when you use `load_data` and when you use `asdata.to_file` in line 70.